pkg/action/env.container.go

```diff
@@ -223,6 +242,20 @@ func (c *containerEnv) Close() error {
 	return c.driver.Close()
 }
 
+func (c *containerEnv) ImageRemove(ctx context.Context, a *Action) {
+	resp, err := c.driver.ImageRemove(ctx, a.ActionDef().Image, types.ImageRemoveOptions{
+		Force:         false,
+		PruneChildren: false,
+	})
+
+	if err != nil {
+		cli.Println(err.Error())
+	} else if resp != nil && resp.Status == types.ImageRemoved {
+		msg := fmt.Sprintf("Image %q was successfuly removed", a.ActionDef().Image)
+		cli.Println(msg)
```

Comment:
You may use directly `cli.Println("Image %q was successfuly removed", a.ActionDef().Image)`